### ISSUE TITLE
Launch-plan for sw-framework & sw-cli

### ISSUE THREAD

**Issue by addyosmani** [GoogleChrome/workbox#111]
**Library Affected**:
*All new libraries*

**Browser & Platform**:
*All browsers*

**Issue or Feature Request Description**:

While we can't publicly comment on I/O dates just yet, I think these are good top-level goals:

- [ ] Launch a stable version of sw-framework and the CLI
- [ ] Publish stable API documentation for both these pieces in the repo
- [ ] Publish developer guides (getting started, migrating from sw-toolbox/precache) on /web
- [ ] Prepare working demo app (could be iFixit or one of the HackerNews apps) using sw-framework

Nice-to-haves:

- [ ] Launch a stable version of the Background Sync library with docs + example
- [ ] Migrate sw-offline-google-analytics to the new Background Sync library

Stretch:

- [ ] Update existing toolbox/precache codelab to use sw-framework 

This is dependent on #110, #109, #134, #169, #198, #202, #206, #205, #173, #164, #135, #195 landing.

**Comment by prateekbh** [comment]
Will it be cool if we can launch a 0.1 version of BGSync library without binary request?
that ways only docs and Request Wrapper remains for me to complete

**Comment by addyosmani** [comment]
> Will it be cool if we can launch a 0.1 version of BGSync library without binary request?
that ways only docs and Request Wrapper remains for me to complete.

👍 Let's do that too.

**Comment by prateekbh** [comment]
FYI: I've done docs, demo and RequestWrapper too.
Some reformatting might be needed though.

**Comment by addyosmani** [comment]
@gauntface @jeffposnick For the Sydney beta, shall I go ahead and move the issues we want to nail down before that release into a milestone we can track? We can do a separate one for I/O once we hear more team feedback in our hackday.

**Comment by gauntface** [comment]
@addyosmani please do
